PRIVACY_POLICY.md

```diff
@@ -0,0 +1,118 @@
+
+### How is this Information Collected
+
+When you send a request of any kind to the Pulsar Backend this information is made available to us through the web request itself and the [User Agent](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/User-Agent) of the request.
+
+### What is Done with this Information
+
+* Your IP address is logged from the Pulsar Backend to aid in later troubleshooting that may need to occur.
+* Your Browser Details and Operating System Details are logged by the service used to Host the Pulsar Backend. But no further action is taken with this information once logged.
+
+### Who has Access to this Information
+
+The only individuals that have access to the above details are Pulsar's Core Admin Team in charge of the Pulsar Backend.
+
+### How Long is this information Kept
+
+The logs that contain the above information is kept in the cloud for 30 days before it is automatically deleted.
+
+### TLDR
+
+* The Pulsar Backend logs your IP Address, and your User-Agent Details of your Web Request when accessing it's service.
+* These logs are kept for 30 days until they are deleted automatically.
+* The only people that have access to these logs are Pulsar's Core Admin Team in charge of the Pulsar Backend.
+* The only reason this data is used is to aid in troubleshooting when/if needed.
+
+## Pulsar User Account 
+
+When you create a user account with Pulsar, to star, and publish packages you provide some details to Pulsar to create the account.
+But it should be made explicit that Pulsar does not store any type of credentials your account at any time.
+
+### What Information is Collected
+
+* GitHub Username
+* GitHub Gravatar URL
+* GitHub `node_id` (Think of your `node_id` like the random number assigned to you when making a user account. This is public information)
+
+### How is this Information Collected
+
+When you sign up with Pulsar, and choose to make an account, this is the data collected to make sure your account works, and is able to protect Packages from malicious
+Actors pretending to be someone they're not.
+
+### What is Done with this Information
+
+This information is used to let you authenticate against the Pulsar Backend, when publishing, deleted, or starring a package.
+
+### Who has Access to this Information
+
+The only people who have access to this information are Pulsar's Core Admin Team in charge of the Backend.
+
+### How Long is this Information Kept
+
+The above details will be kept until either you request deletion of your account, or there is a built in supported method to delete your Pulsar User Account.
+
+### How do I have this Information Deleted
+
+To request removal of the above details feel free to contact the Pulsar Core Admin Team, through any of the [supported methods](https://pulsar-edit.dev/community.html).
+But the recommended method would be to contact @Admins on the [Pulsar Discord](https://discord.gg/7aEbB9dGRT) or via email to admin@pulsar-edit.dev
```

Comment:
Remove "but", perhaps put it above the first one if it is the recommended method. Maybe say instead "for quicker response times, contact through Discord"